relay-dynamic-config/src/global.rs

```diff
@@ -23,6 +25,9 @@ pub struct GlobalConfig {
     /// Quotas that apply to all projects.
     #[serde(skip_serializing_if = "Vec::is_empty")]
     pub quotas: Vec<Quota>,
+    /// Configuration for global inbound filters.
```

Comment:
```suggestion
    /// Configuration for global inbound filters.
    ///
    /// The configuration in the global config is later merged with the more specific
    /// configuration from the project state.
```

Something along the lines that this is only have the truth, what do you think?